openapi.yaml

```diff
@@ -4168,23 +4168,7 @@ paths:
         required: true
       responses:
         '201':
-          description: Created
```

Comment:
deleted custom request object and updated it to be using a reference to Workspace model